lib/modules/datasource/go/releases-goproxy.ts

```diff
@@ -211,10 +215,24 @@ export class GoProxyDatasource extends Datasource {
       .split(newlineRegex)
       .filter(is.nonEmptyStringAndNotWhitespace)
       .map((str) => {
-        const [version, releaseTimestamp] = str.split(regEx(/\s+/));
-        return DateTime.fromISO(releaseTimestamp).isValid
-          ? { version, releaseTimestamp }
-          : { version };
+        let version: string | undefined;
+        let newDigest: string | undefined;
+        let releaseTimestamp: string | undefined;
+        [version, releaseTimestamp] = str.split(regEx(/\s+/));
+
+        if (!DateTime.fromISO(releaseTimestamp).isValid) {
```

Comment:
I think we need to use fromFormat like on datasource metadata change?

Reply:
This part of logic is unchanged, I'm actually not where where/how it's used so preferred to keep it as-is